misc/test_site/device_types/rocket/module_config.json

```diff
@@ -14,6 +14,9 @@
     },
     "discover": {
       "enabled": true
+    },
+    "network": {
```

Comment:
isn't this redundant with the top-level enable? in test_site/module_config.json?

Reply:
Removed network entry in module_config.json